.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: Swatinem/rust-cache@v1
+      # - uses: Swatinem/rust-cache@v1
```

Comment:
Disabling caching on the CI would bloat the times significantly (even up to 30 min at a time), can we revert it? :)